snippets/go/closed_channel.go

```diff
@@ -0,0 +1,27 @@
+package example
+
+import (
+	"log"
+
+	"github.com/breez/breez-sdk-go/breez_sdk"
+)
+
+func PrepareSweep(feeRate uint64) {
+	// ANCHOR: prepare-sweep
+	satPerVbyte := feeRate
+	destinationAddress := "bc1.."
+	req, err := sdk.PrepareSweep(breez_sdk.PrepareSweepRequest{SatPerVbyte: satPerVbyte, ToAdrees: &destinationAddress})
+	if err != nil {
+		log.Fatalf("PrepareSweep error %#v", err)
```

Comment:
Just to be consistent with other examples
```suggestion
		log.Printf("PrepareSweep error %#v", err)
```

Reply:
Good point, added